pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -84,17 +86,26 @@ func New(context *clusterd.Context, clusterInfo *cephclient.ClusterInfo, spec ce
 		clusterInfo: clusterInfo,
 		spec:        spec,
 		rookVersion: rookVersion,
-		Replicas:    1,
+		Replicas:    spec.Mgr.Count,
 		exitCode:    exec.ExitStatus,
 	}
 }
 
 var updateDeploymentAndWait = mon.UpdateCephDeploymentAndWait
 
+// for backward compatibility, default to 1 mgr
+func (c *Cluster) getReplicas() int {
+	replicas := c.Replicas
+	if replicas == 0 {
+		replicas = 1
+	}
+	return replicas
+}
+
 func (c *Cluster) getDaemonIDs() []string {
 	var daemonIDs []string
-	for i := 0; i < c.Replicas; i++ {
-		if i >= 2 {
+	for i := 0; i < c.getReplicas(); i++ {
+		if i >= defaultMaximumNumberOfMgr {
```

Comment:
Since the crd schema allows a max value of 2, we don't need to check this in the code, right?

Reply:
ah thank you, that is right.